src/Model/Filter/Like.php

```diff
@@ -56,6 +57,7 @@ public function process()
         $isMultiValue = is_array($value);
 
         $conditions = [];
+        $colTypes = $this->config('colType') ?: [];
```

Comment:
The `?: []` is not necessary as the default value for `colType` is already set to `[]`. Also the value can be fetched inside `if (!empty($conditions)) {` itself.